### ISSUE TITLE
unbound 1.13.0 crash

### ISSUE THREAD

**Issue by toralf** [NLnetLabs/unbound#432]
the crash and related  is seen here: https://lkml.org/lkml/2021/2/20/108

Few more info for the package here at a stable Gentoo hhrdened server:
```
#  emerge -qpvO unbound
[ebuild   R   ] net-dns/unbound-1.13.0  USE="dnscrypt ecdsa http2 threads -debug -dnstap -ecs -gost (-libressl) -python -redis (-selinux) -static-libs -systemd -test" PYTHON_SINGLE_TARGET="python3_8 -python3_7 -python3_9" 
```

**Comment by gthess** [comment]
There is not a lot of information about the crash at the link but the current master branch includes a couple of fixes that could lead to random crashes; especially when reloading Unbound.

**Comment by gthess** [comment]
No extra information on this and in the meantime a random crash issue was fixed (#612); closing.
